### ISSUE TITLE
Missing keyboard navigation

### ISSUE THREAD

**Issue by TheLastProject** [Pext/Pext#241]
Need to be fixed, suggestions on what keyboard input to use for this would be very welcome.

- [x] The sort order cannot be changed through the keyboard
- [x] Cannot interact with actionables (those Firefox-style yellow alerts) by keyboard
- [ ] Multiline text dialog cannot be submitted by keyboard

**Comment by TheLastProject** [comment]
Multiline can be submitted with Alt+Enter.
